src/components/Login/LoginContainer.tsx

```diff
@@ -160,8 +156,11 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
       lastResendWarningMessageRef.current = response.warning ?? "";
       return true;
     } catch (e) {
-      if (e instanceof LoginError) {
-        showAlert({ ...e.alertProps, onOk: () => resetStage() });
+      if (e instanceof LoginLockedError) {
```

Comment:
Did the error name change?

Reply:
I wanted to only feed the dynamic portion of the message only when it was a `LoginLockedError`, but I have refactored this to feed the dynamic portion regardless of what error it is.